packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -148,8 +146,8 @@ ${chalk.bold('Examples')}
       return this.help()
     }
 
-    if (!args['--preview-feature']) {
-      throw new MigrateDiffNeedsPreviewFeatureFlagError()
+    if (args['--preview-feature']) {
+      console.warn('--preview-feature is deprecated and will be removed in the next major version.')
```

Comment:
I think this can get a nice wording here. Something along the lines that the feature is not a preview feature any more and that the command is not needed any more. I also don't think we would ever really make --preview-feature fail but just remove the warning one day.

Reply:
```suggestion
      logger.warn(`"prisma migrate diff" was in Preview and is now Generally Available.
You can now remove the ${chalk.red('--preview-feature')} flag.`)
```
Something like this maybe, also use our `logger.warn()`, useful for Redwood for example so they can disable the warnings.